src/components/nodes/miscellanious/Preview.vue

```diff
@@ -32,16 +32,19 @@ export default {
       }
     ],
     outputs: [],
-    calculate (input, component) {
-      component.value = input.value
-      if (input.value === undefined) component.type = undefined
-      else if (input.value instanceof BackendImage) component.type = 'image'
-      else component.type = 'other'
+    data: () => ({
+      value: null,
+      type: undefined
+    }),
```

Comment:
This is the instance data. Not to be confused with Vue Component data further down.